test/scss/feature-targeting.scss

```diff
@@ -53,4 +54,9 @@
   @include mdc-typography-overflow-ellipsis($query: mdc-feature-any());
   @include mdc-typography-baseline-top(0, $query: mdc-feature-any());
   @include mdc-typography-baseline-bottom(0, $query: mdc-feature-any());
+
+  // Menu surface
```

Comment:
This section is missing a call to the baseline `mdc-menu-surface` mixin. Also can we insert this after menu rather than at the end?